<launch>
  <arg name="map_file" default="citi.bin"/>
  <arg name="load_map" default="true"/>
  <node name="orb_slam2_mono" pkg="orb_slam2_ros"
      type="orb_slam2_ros_mono" output="screen">

       <remap from="/bebop/image_raw" to="/camera/image_raw" />

       <param name="publish_pointcloud" type="bool" value="true" />
       <param name="publish_pose" type="bool" value="true" />
       <param name="localize_only" type="bool" value="false" />
       <param name="reset_map" type="bool" value="false" />

       <!-- static parameters -->
       <param name="load_map" type="bool" value="$(arg load_map)" />
       <param name="map_file" type="string" value="$(arg map_file)" />
       <param name="settings_file" type="string" value="$(find odom_slam_sensor_fusion)/config/bebop2.yaml" />

       <param name="voc_file" type="string" value="$(find orb_slam2_ros)/orb_slam2/Vocabulary/ORBvoc.txt" />

       <param name="pointcloud_frame_id" type="string" value="map" />
       <param name="camera_frame_id" type="string" value="camera_link" />
       <param name="min_num_kf_in_map" type="int" value="5" />
  </node>
  <node name="remap_camera" pkg="topic_tools"
      type="relay" output="screen" args="/bebop/image_raw /camera/image_raw"/>
</launch>